chromadb/db/impl/sqlite.py

```diff
@@ -34,6 +34,7 @@ def __init__(self, conn_pool: Pool, stack: local):
     @override
     def __enter__(self) -> base.Cursor:
         if len(self._tx_stack.stack) == 0:
+            self._conn.execute("PRAGMA case_sensitive_like = ON")
```

Comment:
Was this meant to be included? It seems unrelated to the rest of the changes.

Reply:
I found this bug while testing the PR, so I added it here. I can split it up in another PR, as it is indeed unrelated.

Reply:
Move to PR or leave it here?